Classes/StackedRoll.lua

```diff
@@ -0,0 +1,103 @@
+---@type GL
+local _, GL = ...;
+
+---@class StackedRoll
+GL.StackedRoll = {
+    _initialized = false,
+};
+local StackedRoll = GL.StackedRoll; ---@type StackedRoll
+
+---@return boolean
+function StackedRoll:_init()
+    GL:debug("StackedRoll:_init");
+
+    if (self._initialized) then
+        return false;
+    end
+
+    -- Bind the appendStackedRollItemInfoToTooltip method to the OnTooltipSetItem event
+    GameTooltip:HookScript("OnTooltipSetItem", function(tooltip)
+        self:appendStackedRollItemInfoToTooltip(tooltip);
+    end);
```

Comment:
The latest update will include itemreftooltip support as well, meaning you need to add the following 3 lines here:

```
ItemRefTooltip:HookScript("OnTooltipSetItem", function(tooltip)
    self:appendStackedRollItemInfoToTooltip(tooltip);
end);
```